scripts/version-check.sh

```diff
@@ -38,6 +38,19 @@ else
     exit 1
 fi
 
+echo "▸ Validating ${CARTHAGE_JSON_SPECIFICATION} version-specific path"
```

Comment:
This almost tripped me up just now, so added a check

Reply:
Awesome!!